---
title: Publications
layout: default
---

<!--[CV as pdf ![CV as pdf]({{ BASE_PATH }}/static/pdf-icon.png)]({{ BASE_PATH }}/static/Townes_curriculum_vitae.pdf)-->

Bold: first or senior author. Italics: coauthor.

### Preprints

* **Quantile normalization of single-cell RNA-seq read counts without unique molecular identifiers**. [BioRxiv 2019](https://www.biorxiv.org/content/10.1101/817031v1). Collaborator: [Rafael Irizarry](http://rafalab.dfci.harvard.edu/) 
* *An exact, unifying framework for region-based association testing in family-based designs, including higher criticism approaches, SKATs, multivariate and burden tests*. [BioRxiv 2019](https://www.biorxiv.org/content/10.1101/815290v1). Collaborators: Julian Hecker, Priyadarshini Kachroo, Jessica Lasky-Su, John Ziniti, Michael H. Cho, Scott T. Weiss, Nan M. Laird, and Christoph Lange
* **Generalized Principal Component Analysis**. [Arxiv 2019](https://arxiv.org/abs/1907.02647).

### Publications

* **Feature Selection and Dimension Reduction for Single Cell RNA-Seq based on a Multinomial Model**. [Genome Biology 2019](https://doi.org/10.1186/s13059-019-1861-6). Collaborators: [Stephanie Hicks](http://www.stephaniehicks.com/), [Martin Aryee](http://aryee.mgh.harvard.edu/), and Rafael Irizarry
* *Temperature Tolerance and Species Range in Mountaintop Salamanders and Their Widespread Competitors*. [Journal of Herpetology](https://www.journalofherpetology.org/doi/abs/10.1670/18-110). Collaborators: [David Marsh](https://home.wlu.edu/~marshd/), Kerry Cotter, Kara Farroni, Katherine McCreary, Rachel Petry, and Joseph Tilghman
* *Family-Based Tests for Associating Haplotypes With General Phenotype Data: Application to Asthma Genetics*. [Genetic Epidemiology 2017](https://onlinelibrary.wiley.com/doi/10.1002/gepi.22094/full). Collaborators: Julian Hecker, Xin Xu, Heide Fier, [Chris Corcoran](http://www.math.usu.edu/people/chriscorcoran_facultypage), [Nan Laird](https://www.hsph.harvard.edu/nan-laird/), and [Christoph Lange](https://www.hsph.harvard.edu/christoph-lange/)
* *Missing Data and Technical Variability in Single-Cell RNA-Sequencing Experiments*. [Biostatistics 2018](https://academic.oup.com/biostatistics/article/doi/10.1093/biostatistics/kxx053/4599254). Collaborators: Stephanie Hicks, Mingxiang Teng, and Rafael Irizarry
* *Predicting Subnational Ebola Virus Disease Epidemic Dynamics from Sociodemographic Indicators*. [PLOS One 2016](http://dx.doi.org/10.1371/journal.pone.0163544). Collaborators: Linda Valeri, Oscar Patterson-Lomba, Yared Gurmu, Akweley Ablorh, Jennifer Bobb, and [Guy Harling](http://guyharling.com/)
* **Seed Dispersal of the genus Leea in forest patches of Bataan, Philippines**. Ecotropica 2010.

### Selected Talks

* **Biostatistics and Data Science as a Career Path**. Biology Department, Washington & Lee University. October 2019
* **Dimension Reduction for Single Cell RNA-Seq based on a Multinomial Model**. Joint Statistical Meetings (Denver, CO). August 2019
* **Feature Selection and Dimension Reduction for Single Cell RNA-Seq based on a Multinomial Model**. UPenn Medical School ([Greene Group](https://www.greenelab.com/)). February 2019
* **Unsupervised Learning for Single Cell Gene Expression**. Harvard Biostatistics Cancer Working Group. September 2018
* **Single Cell Housekeeping Genes and Normalization**. Dana-Farber cBio Seminar. April 2018
* **Varying-Censoring Aware Matrix Factorization for Single Cell RNA-Sequencing**. [Bioconductor](https://www.bioconductor.org/help/course-materials/2017/BioC2017/). July 2017
* **Bayesian Methods for Dependent Data**. Biostatistics 245, Harvard School of Public Health. April 2016. Collaborator: Leah Comment
* **Family Based Association Tests for Rare Variants**. [Channing Division of Network Medicine](http://brighamandwomens.org/research/depts/medicine/channing/default.aspx), Brigham and Women's Hospital. September 2015. Collaborator: Christoph Lange

### Posters

* **Gene Expression Autoencoders**. Biomedical Informatics 707, Harvard Medical School. April 2018. Collaborator: Chinmay Shukla
* **Mixture of Experts Analysis of Infectious Disease Outbreak Characteristics**. Computer Science 281, Harvard School of Engineering and Applied Sciences. December 2015. Collaborator: Carla Marquez-Luna
* **Single Cell RNA-Seq Technical and Biological Confounders**. [Program in Quantitative Genomics Conference](http://www.hsph.harvard.edu/2015-pqg-conference/), Harvard Medical School. November 2015. Collaborators: Mihriban Karaayvaz, Shawn Gillespie, Bradley Bernstein, Leif Ellisen, and Martin Aryee

---

### Class Projects and Other Unpublished Work

#### Variational Inference for Mixtures of Dirichlet Network Distributions
Class project for Harvard Computer Science 282R- Bayesian Nonparametrics.

#### Bayesian Nonparametric Time Series Modeling
with [Jeremiah Zhe Liu](http://jereliu.github.io/)- class project for [MIT 6.882](http://www.tamarabroderick.com/course_6_882.html). We implemented a hierarchical dirichlet process/ hidden markov model switching linear dynamical system in R in a replication of portions of [Emily Fox](http://www.stat.washington.edu/~ebfox/index.html)'s dissertation.

#### Varying-Censoring Aware Matrix Factorization for Single Cell RNA-Sequencing 
[BioRxiv 2017](https://www.biorxiv.org/content/early/2017/07/21/166736). Collaborators: Stephanie Hicks, Martin Aryee, and Rafael Irizarry

#### Network Connectivity and Infectious Disease Modeling
with [JP Onnela](http://www.hsph.harvard.edu/onnela-lab/) and Carla Marquez-Luna
